apps/trading/components/accounts-container/accounts-container.tsx

```diff
@@ -75,3 +84,14 @@ export const AccountsContainer = ({
     </div>
   );
 };
+
+const useAccountStore = create<DataGridSlice>()(
+  persist(
```

Comment:
it can be just `persist(createDataGridSlice, { name: 'vega_fills_store' })`

Reply:
Done